index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <nav class ="main-nav">
+        <div class="logo">
+        <a href="#"><p>Worash Aboche</p></a>  
+        </div>
+         <a href="#"> <img src ="./images/Icon-Menu.svg" alt="menu" class="menu"> </a>
+        </nav>
+        <main>
+            <section  class="head-line">
+                <header>
+                    <h1 id="your-name">Hey There. I am Worash </h1>
+                </header>
+                <p id="profession"> I’am a Software Developer</p>
```

Comment:
- [ ] Please I don't see the goal of this header tag here header tag represents introductory content you can use div here instead.